Config/CMakeIncludes/FindTasmanianMAGMA.cmake

```diff
@@ -0,0 +1,248 @@
+########################################################################
+# FindTasmanianMAGMA.cmake module
```

Comment:
Why `FindTasmanianMAGMA.cmake` and not just `FindMAGMA.cmake`?

Reply:
When MAGMA doesn't come with a "find_package()" every project will make their own. This avoids potential conflicts. Someone else may decide to make a different MAGMA module and search for libraries in a different way.